utils/test/cli_installer_tests.py

```diff
@@ -46,20 +48,20 @@
 variant: %s
 ansible_ssh_user: root
 hosts:
-  - connect_to: master-private.example.com
+  - connect_to: 10.0.0.1
     ip: 10.0.0.1
     hostname: master-private.example.com
     public_ip: 24.222.0.1
     public_hostname: master.example.com
     master: true
     node: true
-  - connect_to: node1-private.example.com
+  - connect_to: 10.0.0.2
```

Comment:
In a previous test fix I added `connect_to` and set it to the hostnames.  Some of the other tests made assumptions that this would be the IP so I'm switching it back.